### ISSUE TITLE
TurtleCoin the decentralized community and how we can make it great (again)

### ISSUE THREAD

**Issue by an3ki** [turtlecoin/meta#147]
Hi Guys :)

Over the last few weeks/months there has become a big divide in the community.
Up until this point we have had 2 or 3 goes at addressing the issue(s) and it wasn't until yesterday it was pointed out that perhaps I am also part of the problem and have not provided solutions to the problem and to open a dialogue to address the problems, well here it is............

This is not a witch hunt, this is not a he said she said, this is an open discussion about the issues surrounding our community and the great discord divide.

We are mostly adults, we all have our inner demons, mental health, family, religion, government and personal issues. Lets separate all of these and fix our broken code like we all know how.

The Issues as I see them ( not saying I am right or wrong )

### **1.  Blanket Market Talk Rule and Exile**
This is/was enforced way to hard in the past and has made a lot of members feel uneasy and leave the community _How can we fix this and welcome them back ?_

### **2. The smartest people have the power**
Sometimes this is great and it works, I don't think this is working anymore and needs to be amended somehow

### **3. The Market server and General server is not friendly**
New users come into either discord and get roasted if they break the rules or don't know rules
We will run out of new users one day and be known as the assholes of crypto

_There are many more issues at hand and hopefully some of you guys can interject and we can come up with a set of guidelines, rules and a fair enforcement program to fix our broken code, We need to be more customer/user friendly like we were and I am to blame as much as anyone else._

**Comment by SoreGums** [comment]
1. this rule is self-evident, explained many times and for you to bring it up shows a lack of understanding and sets up this thread as a troll thread...

2. not sure this is something that can be addressed well, as it implies things that are subjective.

3. market server is not run by the general server people - if there are people in both then that's that, they are in both. people have a choice. being friendly in general server is up to the people in general, I'd encourage people to be welcoming in the #general channel and the no-market rule is usually when things go south, based on what i read in OP seems like you want to market talk and are unhappy that you can't do that in general - to which I refer to 1. above...

I hang out in the development rooms and nothing presented in the original post is represented of what I see in those rooms, I also go into #help frequently too. Will be observing to see how this plays out then...

**Comment by an3ki** [comment]
This is not a troll thread at all!

I have a complete understanding of the no market talk rule and agree with it, the issue is people have been victimized over it and or sent to the market server where they are then roasted and leave the community forever, this looks and feels bad for everyone. I don't have the answer on how to fix it, hopefully together we can fix it

Like it or not both servers are to blame and I created this as a way we can hopefully work on a set of guidelines for both servers to adhere too

**Comment by csd113** [comment]
I can throw my hand in the ring here!

1. Totally agree, the rules and severity of their enforcement made a lot more sense when turtlecoin was young and the market was still mostly full of moon boys looking to get rich quick.  This is a crypto currency and people will mention price at some point, its a valid rule but i have seen members act in ways that a good turtle would not towards new users who dont understand that the rule is more than a guideline.

2. I see this as a positive, the server is run by people who know how these systems work which makes the feel on the server more akin to a linux dev board than a meme coin.

3. The market server is a pretty toxic place, but turtlecoin came from /biz/ and the market room being the only place to talk price as well as having much more lax rules on aggression echoes the /biz/ mentality. Over the last year and a bit the number of active users has declined and those that remain are the founding community as well as community heads, this has created a tight knit group of users who recognize one another, this unfortunately always creates the perception of outwards hostility to new users.

Running a large cryptocurrency community though discord alone has potential for issues, imagine if there was a bitcoin discord with all the devs, it would be chaos. 

So here's my suggestion for fixing these issues:  We should encourage and support the growth of the turtlecoin community on other mediums, the turtlecoin subreddit would be a much better avenue for first time users to go, unfortunately these platforms have been looked at as an off shoot of the discord not a piece of the community and therefor have been very quiet.  We should expand out from just discord to other avenues.

**Comment by SoreGums** [comment]
ok fair enough - people get `hands on hips` real quick which is unhelpful which is part of what happens in the cases I've witnessed - not much to be done about that...

**Comment by Sajo8** [comment]
I very strongly agree with the Blanket Market Talk rule and the Exile rule(?). 

New users who ask an innocent question mildly related to market talk are aggressively and rudely told to go away and go to #market-talk. There is no explanation or any sort of friendly advice given. Naturally, they would be confused and ask or inquire more. Following that, the same point is repeated with more pressure, more aggressively and more insultingly, and then they're exiled. Then, in exile, they're treated like they're some scum of the earth.  
It would be far more acceptable, easy, and effective, if instead:  
a) their question were answered (TurtleCoin's price is x sats as of right now)  
b) told that discussion doesn't happen here (separate market talk from development and community)  
c) any further talk should happen in the other server (in the Market-Talk specific server linked in #market-talk, not in the channel itself)  

Note the "*linked in* #market-talk". Most people, when they come to a cryptocurrency's discord - or just any discord - are looking specifically for help, they don't have hours of time to slowly find their way around. Immediately when they come, they're probably going to ask a question, and it might be market-talk related. Then they're rudely told (as I described above) to go to #market-talk. If that happened to me, I would expect to be able to chat in the linked channel. When it's locked, it'd be natural for them to come back to #general (where they were told to go to #market-talk) and ask why it was locked. There, I've observed many times, they're treated like an idiot: "can you not read?", "try looking at the channel", "<brainlet emote", "<brainlet emote 2>", "<brainlet emote 3>", "kek", "stop being such a f-ing idiot and try reading". Noting the fact that the channel is locked, and exists only to give a direct link to the market talk server is important, and will hopefully help curb that experience for newcomers.

Expanding upon that experience, that will instantly ruin the experience and image of turtlecoin when they join. Like it or not, most people are in cryptocurrencies simply for the money; ignoring "mooning" and "wen lambo", many people analyze price charts and would like to discuss it with others and come to conclusions. I rarely login to twitter but upon doing so, my turtlecoin and cryptocurrency-filled feed was 90% price charts and investing-related stuff. If those investors and analysts are rudely told to shut up and stop being idiot, it will sour their experience, and they will share and spread their experience with others, stifling interest for TurtleCoin and ridding us of our "reputation" of having the "best cryptocurrency community". As aneki mentioned, "We will run out of new users one day and be known as the assholes of crypto".

I understand the people in #general, where the people join, have seen people come in before and ask about market-talk, the same scenario, multiple times, but the newcomers don't know that. People cite the "no market-talk" rule. Newcomers don't know that. Sure, they're told by MEE6 to not market-talk, as well as PM'd, but MEE6 is a meme throughout discord and you should know that. In any server I've joined, MEE6's rules are always disregarded, and no one seems to care. It's different in this server, great, but newcomers shouldn't be expected to know that. They may be complete newcomers to the cryptocurrency community, and a sour experience directly upon joining may jeapardize they ever contributing to the TurtleCoin community or any community in any cryptocurrency, which ruins it for everybody. More people means more ideas, which means more development and cool ideas(like rashedyt, he was once exiled and now look at him).

On the exile topic: a few people who hold the powers to exile often don't spend much time in #general, and are often very trigger-happy. It's felt that as soon as the person has been told "you can't market-talk", they have the rights to exile them. Exiling shouldn't be the first thing you do. Banning should be out of the question, and #exile's purpose is to go away with the need of banning certain people. They both have the same severity I feel, and sometimes I feel being exiled is worse; everyone exiled is treated extremely shittily, pinged incessantly and hurled insults at. This is a larger problem with #exile itself, which I'll come back to later.  
People shouldn't be exiled immediately after making a mistake, or even a few times. It's not easy to ingrain something in people's minds: sometimes I, too, make mistakes and accidentally market-talk. I might realize it, I might not, but regardless, I don't see me being exiled(because I'm "trusted", I assume). We might have to tell them repeatedly, you can't market talk, but that doesn't mean we become more and more aggresive each time we tell them. Additionally, warning them to not market talk, telling to *apologize*(why?), and then *exiling them* when they don't *apologize within an hour* is the most shittiest behavior I've ever seen. Those who exile don't spend much time actually interacting with the community. They usually hang out in the #dev_ channels, where you obiviously won't see this behavior. As @SoreGums demonstrated: "I hang out in the development rooms and nothing presented in the original post is represented of what I see in those rooms,". People who do not have much sense of the community in general and are used to people not market-talking are given the power and authority to exile people, with no oversight or limitations.

About the community in #exile: it needs to be fixed. #exile isn't a place for people to hang out, post inappropriate content and hurl insults. #exile was made instead of banning in order to help those who have violated rules *many many* times (2 times doesn't count) to realize what they've done wrong, and them become good turtles again (what soregums mentioned). I'm going to use @rashedmyt as an example again. He was exiled. He folded a turtle, then asked rock what to do. He told rashed to learn go, and now he's brought the turtle project to new heights. Being rude and mean to the inmates is not the way to be. They were placed in there so that they can further themselves as individuals, rather than be treated like absolute shitholes. People there should be encouraged, and helped.

Citing what @SoreGums said, again, "this rule is self-evident, explained many times and for you to bring it up shows a lack of understanding and sets up this thread as a troll thread...". This behavior/attitude about this issue - dealing with market talk - is a *dealt* issue is completely wrong, and something which inhibits the development and furthering of the whole project. If we have no one to use the project or spread info about the project, the project is useless. Sometimes issues like these are hard and its not easy to come to a conclusion, but they must be dealt with. Thank you @an3ki for starting this

I will most likely have more to add, and will comment more on this post, and edit this, but as of now, this is what I have to say. Thank you, again, @an3ki

**Comment by BrandonT42** [comment]
If I may...

I feel as though the vast majority has *no problem* with how things are done. We have one rule. You break the rule, you're sent to exile. You reconcile, you get out of exile. How the average member reacts to these situations and to new members is *on them*. A majority of the TurtleCoin community has *never broken this rule*, and a majority of new members are welcomed just fine. I feel as though you're hearing a loud minority (folks coming in to make a quick buck and whom can't handle a smack on the wrist) and assuming that's the state of the community.

**Comment by Sajo8** [comment]
"We have one rule. You break the rule, you're sent to exile." That's not the rule. The rule is no market-talk. Someone makes a mistake? That's **fine**. We tell them, nicely, that's not what you do, and help them understand. The TurtleCoin website says "The TurtleCoin community is very welcoming to all users and developers. You won't get shouted at when things break, and we welcome critiques of our work. Please join us in our Discord Chat.". Rudely insulting users when they make a genuine mistake, thinking out server would be generally like all other cryptocurrency servers, is a wrong move to do.

And exiling people is not "a smack on the wrist", as I talked about in my above comment

**Comment by old-greywolf** [comment]
unfortunately, #exile has also been used vigorously buy some moderators because their egos get bruised

**Comment by SoreGums** [comment]
everything being highlighted is not in line with [how-to-be-a-good-turtle](https://medium.com/@turtlecoin/how-to-be-a-good-turtle-20a427028a18) so seems like that is the crux - start being good turtles and remind people to read it...

**Comment by BrandonT42** [comment]
See now you are misunderstanding my semantics. I don't consider mentioning an exchange or coming in to he server and immediately asking what exchanges they can get TurtleCoin at as *breaking* the rule. It's when people are repeatedly warned and continue to do so. That was also not the main point of my message, and I feel as though you are reaching for something to be upset over at the moment. I said my part, and now I am ducking out if this thread. 😀

**Comment by Sajo8** [comment]
The points highlighted in the article are generally not in line with what we're discussing here. The article talks about the way to be a good turtle, the issues being highlighted here (at least, the ones I've highlighted) are about the current problems in our community; what is happening, and what might possibly be a proper way to go about it. Ignoring all of that in favor of a single short post posted over a year ago is, again, the completely wrong behaviour/attitude to have towards the issues being brought up here @SoreGums

**Comment by csd113** [comment]
>A majority of the TurtleCoin community has _never broken this rule_, and a majority of new members are welcomed just fine. I feel as though you're hearing a loud minority (folks coming in to make a quick buck and whom can't handle a smack on the wrist) and assuming that's the state of the community.

Well of course its a minority of people who have problems, but you've gotta know that it is that same loud minority that sours the taste of the outside population.

Crypto is young and a large majority of people in the world now dont have technical skills, those that both dont have them and find themselves in crypto will not look for a community where there is any loud hostility.

**Comment by Sajo8** [comment]
> I don't consider mentioning an exchange or coming in to he server and immediately asking what exchanges they can get TurtleCoin at as breaking the rule. It's when people are repeatedly warned and continue to do so. 

I agree that people should be exiled when they are repeatedly warned not to market-talk, but show a complete disregard for that rule and continue to; #exile exists for a reason. But the attitude towards people making honest mistakes, the frequency of exiling and the treatment of those in #exile are the big issues which need to be addressed (and most likely others I didn't mention in this comment, of course)

> That was also not the main point of my message

I'm sorry, I may have misunderstood the point you were conveying then. Could you re-word it maybe?

**Comment by old-greywolf** [comment]
sajo is that absolutely correct up above when he talks about the behavior of so-called adult grown up people who act like 12 year old adolescents allowed to use foul language for the first time in their lives and they go down to the Exile room and completely make fools of themselves and if they don't have the balls look at themselves in the mirror or to tell their grandkids how great they act down there then they're not really good turtles (in my opinion)

**Comment by SoreGums** [comment]
@Sajo8 	perhaps - however it clearly states to be helpful - being an egomaniac and exiling people over misunderstandings and not knowing how the place operates fits firmly in that article. if anyone reading this has stomped on people and is annoyed at people complaining they really need to put their hand up and ask to be removed from the exile-able group.

When I've been a first party witness to an overstep I've said something and it comes back to that article about being helpful. 

scrolling down that article it also addresses coming in demanding about the price and such
![image](https://user-images.githubusercontent.com/317/60313176-34c40200-99b2-11e9-8d21-900c3e732518.png)

I am agreeing that people being unhelpful and short with newbies is not ideal, being an open community anything more than the good turtle article is asking for unintended consequences. I might not talk in general much, I still scroll it, I have not seen what prompted this issue to be created, recently, I have seen it though... 

asking about price needs to be addressed as market talk, told to go check coin-gecko or the market talk server and that's it. if the person doesn't get it, then a simple:

```
this is a tech focused discord server, if you really need to talk about prices and market activity you need to go somewhere else, start in #market-talk. any technical issues or community building chat is welcome in #help or #dev_marketing and while you're getting situated check out how this project started https://turtlecoin.lol/ (scroll to the The Story) and read this article https://medium.com/@turtlecoin/how-to-be-a-good-turtle-20a427028a18
```

if the person continues to be dense they go to #exile - anyone not capable of stopping for 2mins to process the information coming at them is really getting off on the wrong foot.

**Comment by Sajo8** [comment]
I agree @SoreGums. Generally, people need to be welcoming and understanding with newbies. Quite often, we forget that almost every other crypto discord in the sun allows market talk, and such a strict ban in one server is a rare sight(this does not mean I want to get rid of the rule; I wholeheartedly agree with the rule's existence). It is just that the newcomers are treated very badly when they first make their mistake, as well as the separate but related issues of them being exiled without much thought and their treatment in #exile once exiled

**Comment by crappyrules** [comment]
@brandonlehmann I know nobody else is going to point out the obvious, but they're all talking about you.  Just thought I'd fill you in fam.

**Comment by brandonlehmann** [comment]
> @brandonlehmann I know nobody else is going to point out the obvious, but they're all talking about you. Just thought I'd fill you in fam.

![giphy](https://user-images.githubusercontent.com/20562132/60314042-33052a80-992f-11e9-9f33-5a24dceaa542.gif)

**Comment by old-greywolf** [comment]
... and that veiled threat to me, with the Emoji that said "waiting", I think it had drumming fingers on a table top, told me exactly the kind of person that posted that. Both of us know who you are and if anyone that wants to go in the server and look and hover their mouse they can also see it, I made a comment and out came the threat .... I understood it to be a threat

**Comment by an3ki** [comment]
I vote to remove IBMCD's admin/exile rights

**Comment by Sajo8** [comment]
I feel that's not what we're discussing here @an3ki, let's keep things civilized  and reach a conclusion as a community

**Comment by Sajo8** [comment]
In response to what Rock said and pinned in the discord, so that everything is transparent and recorded:
![rock quote](https://i.imgur.com/qL8Goe3.png)

"@¹² ᵒᶻ ᵖʳᵒᵖʰᵉᵗ You're looking at the wrong issue I feel, seeing you pinned that message. No one is saying market-talk is good or that it should be re-instated. The problem is that people are treated like shit when they accidentally market-talk, not knowing the rule or the severity with which the rule is enforced, that people are exiled too quickly too often, and that people are treated like absolute shitholes in #exile when they don't deserve that treatment. I, and i feel no one else here, has no problem with those who repeatedly abd incessasntly market-talk showing a complete dis-regard for the rules despite being warned many, many, times being exiled.
"

**Comment by japakar** [comment]
I posted, have nothing important to say.  But Ill try.
Thanks!
Maybe add a new room thats not exile that new users can be tagged with by admins if they break the rules right off, that sends them to that 1 room for 5/10 minutes and gives the half page of highlights from the good turtle page.  After 5/10 minutes that tag is auto removed and the member can rejoin everyone, if they break the rule again, exile them and let the kids bash them in that room because obviously they wanted it.

Then if new users are scared away, its because they are stupid and cant follow simple directions given, multiple times.  Not just once! :D

**Comment by ExtraHash** [comment]
Rock likes to tell a story about  how the first week, everyone in the discord got together and had a discussion and a decision was made that everyone was going to give TurtleCoin a run at making it a serious thing.

We've reached a point where having the developers moderate the community is not working. It doesn't scale well: you may all be great developers but being a good developer and being a great discord moderator are very different skillsets. 

Having the devs moderate the community does not scale, and the rough treatment of some users is causing good users to leave.

**Comment by an3ki** [comment]
Anyway, thanks for listening and merry Christmas

**Comment by elkimek** [comment]
Thanks @Sajo8 and @an3ki for bringing this up. 

I like and I understand the market-talk rule after all but I feel it has been exaggerated over and over again. My feelings about present enforcing of the rule (not the rule itself) is that people and the **newcomers** mostly need to put much, much much larger effort in not breaking the rule than focusing on whats important or on what they need.

TL;DR our Discord UX is terrible because of how the rule is enforced.

I remember my first time I came in. It was when Binance announced that listing requests are possible with optional listing fees, that will go to the charity. I just asked if we might do a "community charity fund" of course with the intention being listed there. 

Suddenly the main admin shouted at me **"Get your Binance shit out of here!"**. I was trying to explain what I ment but it got even worse. 

Not sure why I stayed back then. I think I liked Turtlecoin when I made some research and of course the price was nice. I had similar experiences few more times but I think I just got used to it. It's just not the first and the most friendly space in my mind when I want to chat with people about cryptocurrency.

I also know about more people who didn't come back after similar experience and its shame. I used to think that the Turtlecoin community is one of the best in the cryptocurrency space. Now I'm thinking if it didn't become otherwise.

**Comment by LeoStehlik** [comment]
Reading all this, I think we're can identify two discussion points here: 

1. community management: the level of moderation / enforcement of the no-market-talk rule, especially in regard to newcomers. Here I agree with Extrahash - having hard(core) devs moderating the community would not scale and may be a part of the problem.

2. Over-use of exile

Bottom line: we are all people, and have our own understanding what market talk is. If we talk about the two sides of the spectrum: sure, "Wen moon" and other bullshit is a clear market-talk, however wanting to sell something in #merchandise and being asked to remove USD equivalent to TRTL price ... well, there I was really scratching my head and thought maybe this is pushing it too far. 

Two things are required here:
1. common sense when moderating a growing community. That will help to improve the issue #1 
2. use exile as a last resort. Harassment, vulgarities, disrespect, anything against "be a good turtle". Absolutely, jail them in. Clear market talk? Explain, explain and explain. If it does not help, only then exile. 

Two servers policies? Sure thing, it works. 

We got a good place for devs and technology, and a place for market-talk. 

We are not good at addressing the grey zone in-between where technology meets the market and vice-versa; an there we will have to think for the future if/how to cover this grey zone, and whether this is something potentially beneficial for the community/project.
